README.md

```diff
@@ -49,7 +49,7 @@ Available Commands:
 
 Flags:
   -h, --help               Help for gitops
-      --namespace string   Weave GitOps runtime namespace (default "wego-system")
+      --namespace string   The namespace scope for this operation (default "wego-system").
```

Comment:
This is a change I did in a branch. Trying to be super specific about what this flag does.

```suggestion
      --namespace string   The Kubernetes namespace in which Weave GitOps objects will be stored (default "wego-system").
```

Reply:
@jpellizzari specific can be tricky... doing a `get/status` or `remove` isn't a "will be", it's "the objects are here already"

Reply:
@sympatheticmoose Not sure what you mean. We are open to suggestions on the text itself.

Reply:
I gave my suggestion already in the issue ;) I'm saying "will be stored" implies that this is always an action where I am about to store objects which don't yet exist into the given namespace. Which isn't the case for read or remove operations.

Reply:
OK I missed that. @joshri ^^

Reply:
Got it 🌞